Userland/Libraries/LibGfx/ImageFormats/WebPLoader.cpp

```diff
@@ -14,6 +17,34 @@
 
 namespace Gfx {
 
+static ErrorOr<NonnullRefPtr<Bitmap>> rotate_bitmap(NonnullRefPtr<Bitmap> bitmap, Gfx::TIFF::Orientation orientation)
+{
+    if (orientation == Gfx::TIFF::Orientation::Default)
+        return bitmap;
+
+    auto oriented_bmp = TRY(ExifOrientedBitmap::create(orientation, bitmap->size(), bitmap->format()));
+
+    for (int y = 0; y < bitmap->size().height(); ++y) {
+        for (int x = 0; x < bitmap->size().width(); ++x) {
+            auto pixel = bitmap->get_pixel(x, y);
+            oriented_bmp.set_pixel(x, y, pixel.value());
+        }
+    }
+
+    return oriented_bmp.bitmap();
+}
+
+static Gfx::TIFF::Orientation extract_orientation_from_exif(uint8_t const* exif_data, size_t exif_size)
```

Comment:
Maybe this method could take `ReadonlyBytes` instead?

Reply:
I'll try that

Reply:
I haven't found an elegant/nice way to do that :(

Reply:
Done. Ready for review